modules/rtbhouseBidAdapter.md

```diff
@@ -81,11 +81,30 @@ The following steps should be taken to setup Protected Audience for RTB House:
 module documentation.
 
     a. Make sure to enable RTB House bidder to participate in FLEDGE. If there are any other bidders to be allowed for that, add them to the **bidders** array:
-```javascript
-pbjs.setBidderConfig({
-    bidders: ["rtbhouse"],
-    config: {
-        fledgeEnabled: true
-    }
-});
-```
+    ```javascript
+    pbjs.setBidderConfig({
+        bidders: ["rtbhouse"],
+        config: {
+            fledgeEnabled: true
+        }
+    });
+    ```
+
+    b. If you as a publisher have your own [decisionLogicUrl](https://github.com/WICG/turtledove/blob/main/FLEDGE.md#21-initiating-an-on-device-auction)
+    you may utilize it by setting up a dedicated `fledgeConfig` object:
+    ```javascript
+    pbjs.setBidderConfig({
+        bidders: ["rtbhouse"],
+        config: {
+            fledgeEnabled: true,
+            fledgeConfig: {
+                seller: 'https://seller.domain',
+                decisionLogicUrl: 'https://seller.domain/decisionLogicFile.js',
```

Comment:
If a publisher provides this, Would this end up scoring bids from just rtbhouse or potentially other DSPs as well?